internal/converter/converter.go

```diff
@@ -185,6 +206,8 @@ func (c *Converter) convertFile(file *descriptor.FileDescriptorProto) ([]*plugin
 }
 
 func (c *Converter) convert(req *plugin.CodeGeneratorRequest) (*plugin.CodeGeneratorResponse, error) {
+	c.parseGeneratorParameters(req.GetParameter())
```

Comment:
I moved this into `convert` because it was not being called for tests.

Reply:
You just wanted test coverage on parseGeneratorParameters I suppose, fair enough now that it contains regex parsing.